include/glow/Graph/Context.h

```diff
@@ -65,8 +63,17 @@ class Context final {
   /// found.
   Placeholder *getPlaceholderByName(llvm::StringRef name) const;
 
-  /// Inserts the Placeholder-Tensor pair.
-  void insert(Placeholder *P, Tensor &&T);
+  /// Inserts the Placeholder-Tensor pair. If \p overwrite is true then
+  /// deletes an existing tensors for this placeholder if it exists.
```

Comment:
I'd add what happens if overwrite is false and a tensor already exists to this comment.